worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StoragePolicy.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.service.deploy.worker.storage
+
+import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.exception.CelebornIOException
+import org.apache.celeborn.common.internal.Logging
+import org.apache.celeborn.common.metrics.source.AbstractSource
+import org.apache.celeborn.common.protocol.StorageInfo
+
+object StoragePolicy extends Logging {
+  var storageManager: StorageManager = _
+  var conf: CelebornConf = _
+  var createFileOrder: Option[List[String]] = _
+  var evictFileOrder: Option[Map[String, List[String]]] = _
+  var source: AbstractSource = _
+
+  def initlize(
+      conf: CelebornConf,
+      storageManager: StorageManager,
+      abstractSource: AbstractSource): Unit = {
+    this.storageManager = storageManager
+    this.conf = conf
+    this.createFileOrder = conf.workerStoragePolicyCreateFilePolicy
+    this.evictFileOrder = conf.workerStoragePolicyEvictFilePolicy
+    this.source = abstractSource
+  }
+
+  def getEvictedFile(
+      celebornFile: CelebornFile,
+      partitionDataWriterContext: PartitionDataWriterContext): CelebornFile = {
+    if (evictFileOrder.isDefined) {
+      if (evictFileOrder.get.contains(celebornFile.storageType.name())) {
```

Comment:
How about `val order = evictFileOrder.get.get(celebornFile.storageType.name())
if (null != order) {
    return createFile(partitionDataWriterContext, order)
}
null`

Reply:
This change will throw NoSuchElementException.